com.unity.multiplayer.mlapi/Editor/CodeGen/CodeGenHelpers.cs

```diff
@@ -12,8 +12,8 @@
 using Unity.CompilationPipeline.Common.ILPostProcessing;
 using UnityEngine;
 
-#if !UNITY_2019_4_OR_NEWER
-#error MLAPI requires Unity 2019.4 or newer
+#if !UNITY_2020_3_OR_NEWER
+#error MLAPI requires Unity 2020.3 or newer
```

Comment:
this would cause a hard compile error on build immediately if somebody were to put MLAPI into a project older than 2020.3 such as 2020.1, 2019.4 etc.

should we keep this or should we rely on package manager, package.json and other things constraining people to get MLAPI on older versions? @mattwalsh-unity @JesseOlmer @seanstolberg-unity